cross/transmission/Makefile

```
@@ -1,26 +1,24 @@
 PKG_NAME = transmission
-PKG_VERS = 3.00
+PKG_VERS = 4.0.0
 PKG_EXT = tar.xz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://github.com/transmission/transmission/releases/download/$(PKG_VERS)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/zlib cross/openssl cross/curl cross/libevent
+DEPENDS = 
```

out of the build log files, it depends on curl

```suggestion
DEPENDS = cross/curl
```